pipelines/broad/arrays/imputation_beagle/ImputationBeagle.wdl

```diff
@@ -18,7 +18,7 @@ workflow ImputationBeagle {
     File multi_sample_vcf
 
     File ref_dict # for reheadering / adding contig lengths in the header of the ouptut VCF, and calculating contig lengths
-    Array[String] contigs
+    Array[String] contigs # list of possible contigs that will be processed
```

Comment:
I decided not to rename this input variable mostly out of laziness, but if we think it'll be much clearer to rename to something like `valid_contigs`, I'm happy to do that and update the service and other WDLs to match.